src/server/rpc/procedures/MovieDB/MovieDB.js

```diff
@@ -0,0 +1,358 @@
+// This is the MovieDB game RPC. It wraps the web API of themoviedb.org.
+
+'use strict';
+
+var mdb = require('moviedb')(process.env['TMDB_API_KEY']),
```

Comment:
Currently, this causes the server to crash if this env variable is not set. Can you change it to fail gracefully (that is, only disable these features rather than take down the whole server)?